src/Tempest/Framework/Testing/IntegrationTestCase.php

```diff
@@ -18,7 +18,7 @@
 use Tempest\Http\Method;
 use Tempest\Http\Request;
 
-abstract class IntegrationTest extends TestCase
+abstract class IntegrationTestCase extends TestCase
```

Comment:
I think that this better indicates what kind of class this is

Reply:
Let's keep it as a separate PR though :)